### ISSUE TITLE
Looks like this is no longer maintained?

### ISSUE THREAD

**Issue by bkushigian** [pj4533/PokerNowKit#12]
Hey @pj4533, I love all the infrastructure you have but looks like it isn't maintained anymore. I've chatted with a few people about these projects and had a few people ask how hard it would be for me to fork/update. I've started poking around a bit w/ the intent of translating PokerNowKit to python, but before I get too deep I wanted to check to see if you had any plans on maintaining in the future.

**Comment by bkushigian** [comment]
Currently I'd just plan on getting the CLI hand coverter (pn2ps) working. I _might_ take a stab at the hand grabber too, cuz that  looks really cool and I don't know how to do that kind of stuff so it would be fun to learn :)

**Comment by pj4533** [comment]
Yeah I haven't been playing poker lately, so no immediate plans to work on this. I open sourced it so someone else could pick it up if they wanted, so absolutely go for it. The command line pn2ps could be converted to python easily. That is just a straight conversion. The difficult bit is in the converting of the file format. Thats the part that broke all the time, with every update to the PokerNow log format (and why it doesn't work right now).

The hand grabber stuff, is just infrastructure for tricking PokerTracker into thinking you are running PokerStars, so it will enable the HUD. That might be tricky in python, not really sure. They base it all on the name of the window, so if you can get an invisible window to come up, and name it correctly, then the HUD should work too.

**Comment by bkushigian** [comment]
Great, thanks! One of the perks of Python is that there's a bigger user base so maybe more random PRs from active pokernow users?

Anyway, got most of it translated to python and it was pretty easy going.

I might hit the pokernow dev up and see if he can keep me up to date on log format changes. I know they shift over time so it can be tricky.

**Comment by bkushigian** [comment]
Oh, one last question: do you have a pokerstars hand history format spec that you went by, or did you just look at a bunch of examples? I only have ignition hand history files and those seem to be a bit different.

**Comment by pj4533** [comment]
Yeah I just reverse engineered looking at histories I got online, or from errors when I went to import to PokerTracker. A spec would have been nice!

**Comment by bkushigian** [comment]
> A spec would have been nice!

oof, tell me about it

> I just reverse engineered looking at  histories I got online

any leads where I could find some PS hand histories? Found a gist with like two hands in it, but that doesn't cover all the different minutia.

**Comment by pj4533** [comment]
I found that as well...I never found a good source for them. The ones I found Googling (like that gist) were enough to get me going. Then when I ran into a specific case I couldn't figure out, I just imported it into PokerTracker and looked for errors. Not sure how to do concepts that don't exist in PokerStars though (like running out the board multiple times, etc). I stopped working on this prior to that feature being implemented.
